go.mod

```diff
@@ -1,32 +1,32 @@
 module github.com/heetch/avro
 
-go 1.19
+go 1.24
```

Comment:
It will force dependants to this version. 1.23 should be enough, and doesn't prevent dependants to use 1.24.

Or did any of the dependency updates force the bump here?

Reply:
1.23 sounds good